src/resolvers/powerBoostingResolver.ts

```diff
@@ -168,6 +168,6 @@ export class PowerBoostingResolver {
 
   @Query(returns => Number)
   async getTopPowerRank(): Promise<Number> {
-    return getTopPowerRank();
```

Comment:
renameQuery

Reply:
Unless frontend already implemented I think its ok to leave it like this for now. But I leave it up to you.

Reply:
Yeah I afraid of that, so I didnt change the name